docs/releases/1.8.7.txt

```diff
@@ -36,3 +36,8 @@ Bugfixes
   :class:`~django.contrib.postgres.fields.DateTimeRangeField` (:ticket:`24937`).
 
 * Fixed the exact lookup of ``ArrayField`` (:ticket:`25666`).
+
+* Fixed a data loss possibility when using
+  :class:`~django.db.models.PositiveIntegerField` or
+  :class:`~django.db.models.PositiveSmallIntegerField` with
```

Comment:
We could describe this in a bit more detail I think. Something like "when saving values greater than ... on MySQL."